docs/site/themes/template/layouts/partials/download-table.html

```diff
@@ -52,4 +52,9 @@ <h3>{{ .version }}</h3>
             </tbody>
         </table>
     {{ end }}
-{{ end }}
\ No newline at end of file
+    {{ if .ova }}
+        <p>Open virtual Appliance (OVA) 
+        <a href="{{ .ova_page_link }}" title="download page"</a> for this release of Tanzu Community Edition
+        <p>
```

Comment:
Need a closing `</p>` here instead of another opening tag.

Reply:
Fixed.